src/system-image.js

```diff
@@ -53,10 +53,10 @@ var installLatestVersion = options => {
       }),
     adb
       .waitForDevice()
+      .then(() => adb.shell("mount -a || true"))
+      .then(() => utils.log.debug("adb mounted all partitions"))
       .then(() => adb.wipeCache())
       .then(() => utils.log.debug("adb wiped cache"))
-      .then(() => adb.shell("mount -a"))
```

Comment:
I moved this up since if cache was not mounted already, "adb.wipeCache()" would not wipe anything :)